frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunLogs.scss

```
@@ -1,6 +1,8 @@
 .odc-pipeline-run-logs {
   display: flex;
+  flex: 1;
   width: 100%;
+  padding: var(--pf-global--spacer--xl) 0 var(--pf-global--spacer--2xl) 0;
```

Looks like too much space at the bottom.
```suggestion
  padding: var(--pf-global--spacer--xl) 0;
```

---

var(--pf-global--spacer--2xl) is not too much. It is the same as previously we had below logs container. 
` window.innerHeight - this.scrollPane.current.getBoundingClientRect().top - 50`

---

But it's super ugly to have so much padding on the bottom vs the side :(
Console normally has 30px padding in places.
Patternfly uses different steps and therefore the xl is 32px.

---

fixed!